lib/whimsy/asf/podling.rb

```diff
@@ -432,6 +432,8 @@ def self.namesearch
         resolution = issue['fields']['resolution']
         resolution = resolution ? resolution['name'] : 'Unresolved'
         [name, {issue: issue['key'], resolution: resolution}]
+        # we want the podling name search with the largest key
+        break
```

Comment:
There are multiple problems with this:

 * This loop is intended to return a list of ALL resolutions for all names, which is converted later to a hash by name.  A break statement would stop at the first resolution for any name.
* blocks return the value of the last statement in the block.  As that last statement is a break, the end result is that `issues` will be a `nil`.

The `to_h` call below will convert the array to a hash, and the behavior of the conversion is that the last one with any given key wins.  The most expedient solution is to add `.reverse` prior to the call to `.to_h`.  A more comprehensive solution would be replace the `sort_by` to a `group_by` followed by code that selects the entry with the largest issue number in every group.